index.html

```diff
@@ -3173,13 +3163,19 @@ <h3><code>LinkedResource</code> Definition</h3>
 						<tr>
 							<td>
 								<code>
-									<dfn>alternative</dfn>
+									<dfn>alternate</dfn>
 								</code>
 							</td>
-							<td>A reference to a reformulation of the resource in an alternative format. OPTIONAL</td>
-							<td>One or more valid URL strings&#160;[[!url]].</td>
 							<td>
-								<a href="#value-array">Array</a> of <a href="#value-url">URLs</a>
+								<p>References to one or more reformulation(s) of the resource in alternative formats.</p>
+								<p>When specified, <code>encodingFormat</code> indicates the format of the reformulation.</p>
+								<p>OPTIONAL</p>
+							</td>
+							<td>
+								<p>One or more instances of <a href="#publication-link-def"><code>LinkedResource</code></a> objects.</p>
+							</td>
+							<td>
+								<a href="#value-array">Array</a> of <a href="#value-link">Links</a>
 							</td>
```

Comment:
Is the order significant or not? This must be made clear in the spec, see, for example, the definition of reading order (2.7.3.2) if it is not the case.

Reply:
I think the order doesn't matter.